### ISSUE TITLE
INVALID_ARGUMENT: Failed: There was an error while trying to create this media item

### ISSUE THREAD

**Issue by BILL-HU-2021** [ylexus/jiotty-photos-uploader#129]
First of all, thank you to the creator for providing this good tool.

I have a question. When using version 2.0.5 to upload MP4 files, some files fail while others succeed. I have confirmed that there are no discernible differences between the successful and failed files, and the failed files are not particularly large. The error log is
"**_INVALID_ARGUMENT: Failed: There was an error while trying to create this media item_**"

Despite multiple retries, the issue persists. I also attempted the upload on the next day, but unfortunately  it still doesn’t work. Additionally, I have verified that there is sufficient available space in my google drive.

**Comment by ylexus** [comment]
Hi, what happens if you try to upload one of these "bad" files manually via Google Photos web?

Also, is it always same files that fail?

Finally, if you can't figure out what's wrong with them please send me the application logs - they are in your home directory -> .jiottyphotosuploader. Are you running it on Windows?

**Comment by BILL-HU-2021** [comment]
Thanks..
While using the web interface directly seems to work fine, I analyzing the log files revealed that these files were previously uploaded and are now being ignored during the upload process. Additionally, they are not being added to the album.

Is it possible to prevent these files from being ignored during upload or to only add them to the album, I have try to change the upload mode settings. However, even after selecting a different option, the issue persists.

**Comment by ylexus** [comment]
I suspect they have been previously uploaded manually, not via the app. If you look at the wiki, this is one of the Google Photos API limitations. When you look at one of these in Google Photos web and click on "info" or "details" - it shows who uploaded and when. If it was jiotty it will say so. What does it show?

**Comment by BILL-HU-2021** [comment]
Yes, but it not manually uploaded, it upload by "Google Backup and Sync" APP

**Comment by ylexus** [comment]
OK, so this would explain it. The key is that it was not uploaded by Jiotty. The public Google Photos API restricts an API user from modifying or re-uploading anything that was not uploaded by the same app. Maybe I could have improved an error message to add this as a possibly explanation, but ultimately this is the reason.

Also it is very unlikely that I will be releasing any new versions unless there is a strong demand. It costs money to keep developer certificates that are needed to release new version as signed/verified apps.

**Comment by ylexus** [comment]
Actually, can you confirm that you have this option set as highlighted?

<img width="840" alt="image" src="https://github.com/ylexus/jiotty-photos-uploader/assets/6009806/67f87362-bfa0-4b08-afbd-0d3a1d5a6bae">

**Comment by BILL-HU-2021** [comment]
I have tried two methods as above and the result is the same.

Although there will be no new version, I still appreciate your contribution. Since PICASA went offline, I have looked for a lot of software, and finally your software can solve my problem. Thank you very much.

Your app is very good
